java/code/src/com/redhat/rhn/domain/errata/Cve.java

```diff
@@ -48,4 +52,28 @@ public void setName(String nameIn) {
     public String getName() {
         return name;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public boolean equals(final Object other) {
+        if (!(other instanceof Cve)) {
+            return false;
+        }
+        Cve castOther = (Cve) other;
+        return new EqualsBuilder()
```

Comment:
I'm not sure about this equals implementation. If the names are different is object equal?
We should not confuse the same DB entry with the Java object being equals. Hibernate uses the equals method to check if an object needs to be flushed to the database, so I think we need to also compare the name here.

Reply:
Done.